lib/index.js

```diff
@@ -16,30 +16,45 @@ var Yandex = module.exports = integration('Yandex Metrica')
   .assumesPageview()
   .global('yandex_metrika_callbacks')
   .global('Ya')
+  .option('type', 0)
   .option('counterId', null)
   .option('clickmap', false)
   .option('webvisor', false)
+  .option('trackHash', false)
+  .option('trackLinks', false)
+  .option('accurateTrackBounce', false)
```

Comment:
I left it `false` for exact same reason - that's a default value, but enabling accurate bounce rate should only positively impact your metrics. That was the main goal of this PR